### PR TITLE
Update svg icons

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Currently, `nns-dapp` icons are sourced from different libraries, resulting in inconsistencies in style and line thickness. This update standardises most of the icons to ensure they match the overall design and maintain a consistent line thickness.

Source: https://www.figma.com/design/qPpcyrRcqBHuHsMHLa3Rci/NNS-Icons?node-id=27-27&t=R1eZMFLMh1q1uXpZ-0
Demo: https://qsgjb-riaaa-aaaaa-aaaga-cai.mstr-ingress.devenv.dfinity.network/

# Changes

- Replace several icon SVGs (sizes preserved).
- Added `strokeWidth` property to the `IconSouth` to make it more prominent in the sorted table.
- Update affected screenshots.

# Screenshots

## Icons that were updated

image

## Before

image

## After

image

## Sync animation after update

https://github.com/dfinity/gix-components/assets/98811342/2b577bbd-f0c1-455b-bf67-b9ab2a7a139a

## Sorting icons

image

image

### CODE REVIEW

**Review by peterpeterparker — COMMENTED** [review]
LGTM, thanks

Notes:

- Is maybe the icon "lock open" actually not open enough? Based on the screenshots of the PR, I find it hard to distinguish which one is open, which one is closed.

- For the future, should we setup an E2E screenshot test for the icon page?

**Review by peterpeterparker — APPROVED** [review]

**Comment by mstrasinskis** [comment]
> LGTM, thanks
> 
> Notes:
> 
> * Is maybe the icon "lock open" actually not open enough? Based on the screenshots of the PR, I find it hard to distinguish which one is open, which one is closed.

Thank you. It's not the first mentioning that the `lock open` icon isn't distinguishable enough, so it’s definitely worth updating. 

image


> * For the future, should we setup an E2E screenshot test for the icon page?

Good point.

File: src/lib/icons/IconLockOpen.svelte [hunk 1]
```diff
@@ -8,18 +8,39 @@
 <svg
   width={size}
   height={size}
-  viewBox="0 0 20 20"
+  viewBox="0 0 21 21"
```
Comment (dskloetd) [inline src/lib/icons/IconLockOpen.svelte:11]: If I copy SVG from Figma, I get:
```
<svg width="21" height="21" viewBox="0 0 21 21" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M6.70651 8.00232H15.0373C16.0032 8.00232 16.7865 8.78561 16.7865 9.75232V15.5857C16.7865 16.5524 16.0032 17.3357 15.0373 17.3357H6.70651C5.74065 17.3357 4.95728 16.5524 4.95728 15.5857V9.75232C4.95728 8.78561 5.74065 8.00232 6.70651 8.00232Z" stroke="white" stroke-width="1.5"/>
<path d="M13.6492 7.25232V4.83899C13.6492 3.39585 14.8191 2.22595 16.2622 2.22595V2.22595C17.7053 2.22595 18.8752 3.39585 18.8752 4.83899V5.09517" stroke="white" stroke-width="1.5" stroke-linecap="round"/>
<path d="M10.8699 12.6691H10.8749" stroke="white" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
</svg>
```
This has both a size and viewBox of 21.
But then if we use it with our default of size 20, but keep the viewBox of 21, then it ends up being smaller than intended, right?

I'm checking this with Artem as well.

Comment (mstrasinskis) [reply src/lib/icons/IconLockOpen.svelte]: Correct, because of the clip-path is 20x20 px.

File: src/lib/icons/IconSouth.svelte
Comment (dskloetd) [inline src/lib/icons/IconSouth.svelte:1]: This icon is used for the sorting direction in the neurons table and became almost invisible.
Can you leave this one alone and add a new icon file with the new svg?
Then we can separately see how to deal with the sorting icon.

File: src/lib/icons/IconLockOpen.svelte (hunk 1 above)
Comment (mstrasinskis) [reply src/lib/icons/IconLockOpen.svelte]: The icons were updated, so there is no more viewBox 21x21.

File: src/lib/icons/IconSouth.svelte
Comment (mstrasinskis) [reply src/lib/icons/IconSouth.svelte:1]: The `IconSouth` was updated for better visibility.

**Pull request merged**